apps/condo/domains/marketplace/schema/Invoice.test.js

```diff
@@ -1603,5 +1603,51 @@ describe('Invoice', () => {
                 messageForUser: 'api.marketplace.invoice.error.PublishingWithoutDefinedPricesForbidden',
             })
         })
+
+        test('can create more then 2 invoice with different ticket', async () => {
+            const createInvoiceWithTicket = async () => {
```

Comment:
```suggestion
            const createInvoiceWithTicket = async (client) => {
```
Please pass `client` here. I tried for a long time to understand where this variable comes from - but it turns out it is declared below)

Reply:
Maybe move this function outside the test? It may be needed in other tests too

Reply:
This is a local simplification; in other tests, the process of creating entities for an invoice uses only test utilities like 'createTest...'. I simplified it here just to avoid a long duplication